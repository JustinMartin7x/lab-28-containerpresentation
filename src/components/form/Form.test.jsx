import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Form from '../form/Form'

describe('Article item component', () => {
  afterEach(() => cleanup());
  it ('renders an article', () => {
    const { asFragment } = render(<Form
      setArticles={() => {}}
      setLoading={() => {}}
    />);

    expect(asFragment()).toMatchSnapshot();
  });
});
