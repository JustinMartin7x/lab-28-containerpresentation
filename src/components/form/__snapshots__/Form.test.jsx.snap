// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article item component renders an article 1`] = `
<DocumentFragment>
  <form>
    <input
      placeholder="Search"
      type="text"
      value=""
    />
    <button
      type="submit"
    >
      Search
    </button>
  </form>
</DocumentFragment>
`;
